kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -23,6 +23,7 @@
 import io.fabric8.kubernetes.client.internal.SSLUtils;
 import io.fabric8.kubernetes.client.utils.Utils;
 import io.sundr.builder.annotations.Buildable;
+import org.apache.commons.lang.StringUtils;
```

Comment:
Ooo good point. Can't have this dep - it's caused issues in the past for adoption/integration. It's only one simple method that we can dupe in if there's no other way around it.